Tests/js/views/fields/assets/ez-relation-editview-tests.js

```diff
@@ -0,0 +1,419 @@
+/*
+ * Copyright (C) eZ Systems AS. All rights reserved.
+ * For full copyright and license information view LICENSE file distributed with this source code.
+ */
+YUI.add('ez-relation-editview-tests', function (Y) {
+    var viewTest, registerTest, getFieldTest, getEmptyFieldTest;
+
+    viewTest = new Y.Test.Case({
+        name: "eZ Relation View test",
+
+        _getFieldDefinition: function (required) {
+            return {
+                isRequired: required
+            };
+        },
+
+        setUp: function () {
+            this.destinationContent = new Y.Mock();
+            this.destinationContentToJSON = {anythingJSONed: 'somethingJSONed'};
+            Y.Mock.expect(this.destinationContent, {
+                method: 'toJSON',
+                returns: this.destinationContentToJSON
+            });
+            this.fieldDefinitionIdentifier= "niceField";
+            this.fieldDefinition = {
+                fieldType: "ezobjectrelation",
+                identifier: this.fieldDefinitionIdentifier,
+                isRequired: false
+            };
+            this.field = {fieldValue: {destinationContentId: 45}};
+            this.jsonContent = {};
+            this.jsonContentType = {};
+            this.jsonVersion = {};
+            this.content = new Y.Mock();
+            this.version = new Y.Mock();
+            this.contentType = new Y.Mock();
+            Y.Mock.expect(this.content, {
+                method: 'toJSON',
+                returns: this.jsonContent
+            });
+            Y.Mock.expect(this.version, {
+                method: 'toJSON',
+                returns: this.jsonVersion
+            });
+            Y.Mock.expect(this.contentType, {
+                method: 'toJSON',
+                returns: this.jsonContentType
+            });
+
+            this.view = new Y.eZ.RelationEditView({
+                container: '.container',
+                field: this.field,
+                fieldDefinition: this.fieldDefinition,
+                content: this.content,
+                version: this.version,
+                contentType: this.contentType,
+                destinationContent: this.destinationContent,
+                destinationContentId: null,
```

Comment:
this attribite is readOnly, so this line has no effect. In addition, it's a bit weird to instantiate the view like that, `destinationContent` and `destinationContentId` are not supposed to be passed at the build time. To get closer to the real usage of this view, you should write somewhere `this.view.set('destinationContent', destinationContent);`